### ISSUE TITLE
 Cannot connect to host twitter.com:443 ssl:True 

### ISSUE THREAD

**Issue by charujing** [twintproject/twint#1057]
Dear all
How to solve this problem:
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host twitter.com:443 ssl:True 
Thanks
rujing

**Comment by himanshudabas** [comment]
Can you please provide more information?
Like you environment, system you are running on, your script etc?

**Comment by yibudak** [comment]
@himanshudabas here is not a good place to ask but any information about rate limitation for Twint? I get rate limitation error after 250 requests in 15 mins. Any way to go around that?

**Comment by himanshudabas** [comment]
@yibudak 
I don't think it should rate limit you for 250 requests.
can you share the script that you are running?

**Comment by charujing** [comment]
> Can you please provide more information?
> Like you environment, system you are running on, your script etc?

Thank you for your reply. I was running the "twint -g="48.880048,2.385939,1km" -o file.csv --csv"
====
Windows 10 professional,  Python 3.8.3 (default, Jul  2 2020, 17:30:36) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Below is packages of python.
====
Requirement already satisfied: twint in c:\programdata\anaconda3\lib\site-packages (2.1.20)
Requirement already satisfied: geopy in c:\programdata\anaconda3\lib\site-packages (from twint) (2.0.0)
Requirement already satisfied: aiohttp in c:\programdata\anaconda3\lib\site-packages (from twint) (3.7.3)
Requirement already satisfied: pysocks in c:\programdata\anaconda3\lib\site-packages (from twint) (1.7.1)
Requirement already satisfied: pandas in c:\programdata\anaconda3\lib\site-packages (from twint) (1.0.5)
Requirement already satisfied: aiohttp-socks in c:\programdata\anaconda3\lib\site-packages (from twint) (0.5.5)
Requirement already satisfied: googletransx in c:\programdata\anaconda3\lib\site-packages (from twint) (2.4.2)
Requirement already satisfied: schedule in c:\programdata\anaconda3\lib\site-packages (from twint) (0.6.0)
Requirement already satisfied: fake-useragent in c:\programdata\anaconda3\lib\site-packages (from twint) (0.1.11)
Requirement already satisfied: cchardet in c:\programdata\anaconda3\lib\site-packages (from twint) (2.1.7)
Requirement already satisfied: elasticsearch in c:\programdata\anaconda3\lib\site-packages (from twint) (7.10.0)
Requirement already satisfied: aiodns in c:\programdata\anaconda3\lib\site-packages (from twint) (2.0.0)
Requirement already satisfied: beautifulsoup4 in c:\programdata\anaconda3\lib\site-packages (from twint) (4.9.1)
Requirement already satisfied: geographiclib<2,>=1.49 in c:\programdata\anaconda3\lib\site-packages (from geopy->twint) (1.50)
Requirement already satisfied: async-timeout<4.0,>=3.0 in c:\programdata\anaconda3\lib\site-packages (from aiohttp->twint) (3.0.1)
Requirement already satisfied: attrs>=17.3.0 in c:\programdata\anaconda3\lib\site-packages (from aiohttp->twint) (19.3.0)
Requirement already satisfied: multidict<7.0,>=4.5 in c:\programdata\anaconda3\lib\site-packages (from aiohttp->twint) (5.1.0)
Requirement already satisfied: chardet<4.0,>=2.0 in c:\programdata\anaconda3\lib\site-packages (from aiohttp->twint) (3.0.4)
Requirement already satisfied: yarl<2.0,>=1.0 in c:\programdata\anaconda3\lib\site-packages (from aiohttp->twint) (1.6.3)
Requirement already satisfied: typing-extensions>=3.6.5 in c:\programdata\anaconda3\lib\site-packages (from aiohttp->twint) (3.7.4.2)
Requirement already satisfied: python-dateutil>=2.6.1 in c:\programdata\anaconda3\lib\site-packages (from pandas->twint) (2.8.1)
Requirement already satisfied: numpy>=1.13.3 in c:\programdata\anaconda3\lib\site-packages (from pandas->twint) (1.18.5)
Requirement already satisfied: pytz>=2017.2 in c:\programdata\anaconda3\lib\site-packages (from pandas->twint) (2020.1)
Requirement already satisfied: python-socks[asyncio]>=1.0.1 in c:\programdata\anaconda3\lib\site-packages (from aiohttp-socks->twint) (1.1.1)
Requirement already satisfied: requests in c:\programdata\anaconda3\lib\site-packages (from googletransx->twint) (2.24.0)
Requirement already satisfied: certifi in c:\programdata\anaconda3\lib\site-packages (from elasticsearch->twint) (2020.6.20)
Requirement already satisfied: urllib3<2,>=1.21.1 in c:\programdata\anaconda3\lib\site-packages (from elasticsearch->twint) (1.25.9)
Requirement already satisfied: pycares>=3.0.0 in c:\programdata\anaconda3\lib\site-packages (from aiodns->twint) (3.1.1)
Requirement already satisfied: soupsieve>1.2 in c:\programdata\anaconda3\lib\site-packages (from beautifulsoup4->twint) (2.0.1)
Requirement already satisfied: idna>=2.0 in c:\programdata\anaconda3\lib\site-packages (from yarl<2.0,>=1.0->aiohttp->twint) (2.10)
Requirement already satisfied: six>=1.5 in c:\programdata\anaconda3\lib\site-packages (from python-dateutil>=2.6.1->pandas->twint) (1.15.0)
Requirement already satisfied: cffi>=1.5.0 in c:\programdata\anaconda3\lib\site-packages (from pycares>=3.0.0->aiodns->twint) (1.14.0)
Requirement already satisfied: pycparser in c:\programdata\anaconda3\lib\site-packages (from cffi>=1.5.0->pycares>=3.0.0->aiodns->twint) (2.20)
====

**Comment by himanshudabas** [comment]
@charujing 
Are you able to open twitter on a browser?

**Comment by charujing** [comment]
> @charujing
> Are you able to open twitter on a browser?
> On the same system you are running twint on?
> 
> It seems that twitter is blocking your IP.
> Take a look at #783, and if you search the issues section, you'll find this same issue being asked several times. Please go through them.

Yes, I can open twitter on Chrome. And I ran the twint on the same windows system.
I used  a public available vpn. Does that mean I need an paid VPN?

**Comment by h4m5t** [comment]
> > @charujing
> > Are you able to open twitter on a browser?
> > On the same system you are running twint on?
> > It seems that twitter is blocking your IP.
> > Take a look at #783, and if you search the issues section, you'll find this same issue being asked several times. Please go through them.
> 
> Yes, I can open twitter on Chrome. And I ran the twint on the same windows system.
> I used a public available vpn. Does that mean I need an paid VPN?

can you tell me how did you solved this problem?
